connectorx-python/connectorx/tests/test_read_sql.py

```diff
@@ -13,6 +13,25 @@ def postgres_url() -> str:
     return conn
 
 
+@pytest.fixture(scope="module")  # type: ignore
```

Comment:
maybe we can make mysql related test into a separated file? test_read_sql is more like using postgres to test more common functionalities like partition, supported query types.